common/big.go

```diff
@@ -39,3 +39,29 @@ var (
 	Num32 = uint256.NewInt().SetUint64(32)
 	Num35 = uint256.NewInt().SetUint64(35)
 )
+
+func SetBytesRightPadded(z *uint256.Int, buf []byte, length int) *uint256.Int {
```

Comment:
This is the variation of this, right? https://github.com/holiman/uint256/blob/master/uint256.go#L26

Reply:
It is.